### PR TITLE
Added a bunch more windows reg persistence keys

### PR DESCRIPTION
**Pull request by liamjm**
Reviewer: @destijl, @joachimmetz

[Review on Reviewable](https://reviewable.io/reviews/forensicartifacts/artifacts/115)

### CODE REVIEW

File: definitions/windows.yaml [hunk 1]
```diff
@@ -228,6 +286,15 @@ sources:
 supported_os: [Windows]
 urls: ['https://github.com/libyal/winreg-kb/blob/master/documentation/Programs%20Cache%20values.asciidoc']
 ---
+name: RegistryShellSpawning
+doc: Executed every time this file type is opened, should be "%1 %*".
+sources:
+- type: REGISTRY_KEY
+  attributes:
+    keys: ['HKEY_LOCAL_MACHINE\Software\Classes\\*\shell\open\command']
```
Comment (destijl) [inline definitions/windows.yaml]: looks like \\ typo?

**Comment by destijl** [comment]
Great stuff thanks, just one small typo.

**Comment by liamjm** [comment]
Removed extra slash.
PTAL.

File: definitions/windows.yaml [hunk 2]
```diff
@@ -443,6 +594,17 @@ labels: [Software]
 supported_os: [Windows]
 urls: ['http://www.forensicswiki.org/wiki/Windows_Desktop_Search']
 ---
+name: WindowsSecurityProviders
+doc: Security Providers DLLs
+sources:
+- type: REGISTRY_VALUE
+  attributes:
+    key_value_pairs: [{key: '%%current_control_set%%\Control\SecurityProviders', value: 'SecurityProviders'}]
```
Comment (joachimmetz) [inline definitions/windows.yaml]: HKEY_LOCAL_MACHINE\SYSTEM\%%current_control_set%%

Comment (liamjm) [inline definitions/windows.yaml]: I copied this from existing examples. Once @destijl provides an answer, I'll update if required.

**Comment by destijl** [comment]
Yep, this is correct on my machine `%%current_control_set%%` is:

`HKEY_LOCAL_MACHINE\SYSTEM\ControlSet001`

https://github.com/google/grr/blob/master/proto/knowledge_base.proto#L293

**Comment by joachimmetz** [comment]
So that does not match the Windows behavior since CurrentControlSet should be ControlSet001

**Pull request merged**